sklearn/metrics/_parallel_reductions.pyx

```diff
@@ -0,0 +1,1162 @@
+# cython: language_level=3
+# cython: cdivision=True
+# cython: boundscheck=False
+# cython: wraparound=False
+# cython: profile=False
+# cython: linetrace=False
+# cython: initializedcheck=False
+# cython: binding=False
+# distutils: define_macros=CYTHON_TRACE_NOGIL=0
+
+import numpy as np
+cimport numpy as np
+cimport openmp
```

Comment:
I need this import to use `openmp.omp_get_thread_num`.

Probably a helper should be created in `utils._openmp_helpers` used instead for scikit-learn version built without OpenMP.

Reply:
Done in 47887ea49056f88e0c9a023aacc35849544eb7b3.